File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/ChannelInfo.java
package com.ctrip.ccard.creditcard.vcc.bean;

/**
 * Description:
 */
public class ChannelInfo {
    /**
     * 卡通道类型
     */
    private String channelType;

    /**
     * 卡通道总限额
     */
    private String channelCardCreditLimitAmt;

    /**
     * 卡通道总限额剩余额度
     */
    private String channelRemainCardCreditLimitAmt;

    /**
     * 卡通道总授信额度
     */
    private String channelAuthLimitAmt;

    /**
     * 卡通道总授信额度剩余额度
     */
    private String channelRemainAuthLimitAmt;

    /**
     * 卡通道限额币种
     */
    private String channelCurrencyCodeOfLimitAmt;

    public String getChannelType() {
        return channelType;
    }

    public void setChannelType(String channelType) {
        this.channelType = channelType;
    }

    public String getChannelCardCreditLimitAmt() {
        return channelCardCreditLimitAmt;
    }

    public void setChannelCardCreditLimitAmt(String channelCardCreditLimitAmt) {
        this.channelCardCreditLimitAmt = channelCardCreditLimitAmt;
    }

    public String getChannelRemainCardCreditLimitAmt() {
        return channelRemainCardCreditLimitAmt;
    }

    public void setChannelRemainCardCreditLimitAmt(String channelRemainCardCreditLimitAmt) {
        this.channelRemainCardCreditLimitAmt = channelRemainCardCreditLimitAmt;
    }

    public String getChannelAuthLimitAmt() {
        return channelAuthLimitAmt;
    }

    public void setChannelAuthLimitAmt(String channelAuthLimitAmt) {
        this.channelAuthLimitAmt = channelAuthLimitAmt;
    }

    public String getChannelRemainAuthLimitAmt() {
        return channelRemainAuthLimitAmt;
    }

    public void setChannelRemainAuthLimitAmt(String channelRemainAuthLimitAmt) {
        this.channelRemainAuthLimitAmt = channelRemainAuthLimitAmt;
    }

    public String getChannelCurrencyCodeOfLimitAmt() {
        return channelCurrencyCodeOfLimitAmt;
    }

    public void setChannelCurrencyCodeOfLimitAmt(String channelCurrencyCodeOfLimitAmt) {
        this.channelCurrencyCodeOfLimitAmt = channelCurrencyCodeOfLimitAmt;
    }
}

File: php-sdk/src/triplink/vcc/card/client/query/ChannelInfo.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/4
 * Time: 11:35
 */
namespace triplink\vcc\card\client\query;


class ChannelInfo
{
    /**
     * 卡通道类型
     */
    var $channelType;

    /**
     * 卡通道总限额
     */
    var $channelCardCreditLimitAmt;

    /**
     * 卡通道总限额剩余额度
     */
    var $channelRemainCardCreditLimitAmt;

    /**
     * 卡通道总授信额度
     */
    var $channelAuthLimitAmt;

    /**
     * 卡通道总授信额度剩余额度
     */
    var $channelRemainAuthLimitAmt;

    /**
     * 卡通道限额币种
     */
    var $channelCurrencyCodeOfLimitAmt;

    /**
     * @return mixed
     */
    public function getChannelType()
    {
        return $this->channelType;
    }

    /**
     * @param mixed $channelType
     */
    public function setChannelType($channelType): void
    {
        $this->channelType = $channelType;
    }

    /**
     * @return mixed
     */
    public function getChannelCardCreditLimitAmt()
    {
        return $this->channelCardCreditLimitAmt;
    }

    /**
     * @param mixed $channelCardCreditLimitAmt
     */
    public function setChannelCardCreditLimitAmt($channelCardCreditLimitAmt): void
    {
        $this->channelCardCreditLimitAmt = $channelCardCreditLimitAmt;
    }

    /**
     * @return mixed
     */
    public function getChannelRemainCardCreditLimitAmt()
    {
        return $this->channelRemainCardCreditLimitAmt;
    }

    /**
     * @param mixed $channelRemainCardCreditLimitAmt
     */
    public function setChannelRemainCardCreditLimitAmt($channelRemainCardCreditLimitAmt): void
    {
        $this->channelRemainCardCreditLimitAmt = $channelRemainCardCreditLimitAmt;
    }

    /**
     * @return mixed
     */
    public function getChannelAuthLimitAmt()
    {
        return $this->channelAuthLimitAmt;
    }

    /**
     * @param mixed $channelAuthLimitAmt
     */
    public function setChannelAuthLimitAmt($channelAuthLimitAmt): void
    {
        $this->channelAuthLimitAmt = $channelAuthLimitAmt;
    }

    /**
     * @return mixed
     */
    public function getChannelRemainAuthLimitAmt()
    {
        return $this->channelRemainAuthLimitAmt;
    }

    /**
     * @param mixed $channelRemainAuthLimitAmt
     */
    public function setChannelRemainAuthLimitAmt($channelRemainAuthLimitAmt): void
    {
        $this->channelRemainAuthLimitAmt = $channelRemainAuthLimitAmt;
    }

    /**
     * @return mixed
     */
    public function getChannelCurrencyCodeOfLimitAmt()
    {
        return $this->channelCurrencyCodeOfLimitAmt;
    }

    /**
     * @param mixed $channelCurrencyCodeOfLimitAmt
     */
    public function setChannelCurrencyCodeOfLimitAmt($channelCurrencyCodeOfLimitAmt): void
    {
        $this->channelCurrencyCodeOfLimitAmt = $channelCurrencyCodeOfLimitAmt;
    }


}

?>